shared/actions/chat.js

```diff
@@ -172,11 +190,29 @@ function * _postMessage (action: PostMessage): SagaGenerator<any, any> {
       message: new HiddenString(action.payload.text.stringValue()),
       followState: 'You',
     }
+
+    // Should we add a timestamp before our new message?
+    // TODO short-term if we haven't seen this in the conversation list we'll refresh the inbox. Instead do an integration w/ gregor
+    const conversationStateSelector = (state: TypedState) => state.chat.get('conversationStates', Map()).get(conversationIDKey)
```

Comment:
how do you get into this state? you're making messages in the cli but not on the chat screen?